src/models/Alunos.js

```diff
@@ -24,17 +24,27 @@ export async function selecionarAluno(req, res) {
 export async function cadastrarAluno(req, res) {
     let aluno = req.body;
     openDb().then(db => {
-        db.run("INSERT INTO Aluno (nome, cpf, numero_matricula, sala_id) VALUES (?,?,?,?)", [aluno.nome, aluno.cpf, aluno.numero_matricula, aluno.sala_id])
+        db.get("SELECT 1 FROM Aluno WHERE cpf=? OR numero_matricula=?", [aluno.cpf, aluno.numero_matricula]).then(existe => {
+            if (!!existe === true) {
```

Comment:
!!existe é o próprio existe, não? 

além disso, se vc quer validar se é true, basta fazer assim if(existe) { } , se existe for true ele entra no if, se for false ele passa direto.